src/Components/Users/Profile/Profile.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { showLoader, hideLoader } from '../../../actions/loader.js';
+import { getBoughtSeatsByUserAsync } from '../../../actions/seats.js';
+import { checkAuthorizationAsync, logOut } from '../../../actions/users.js';
+import Loader from '../../Loader/Loader.js';
+import Ticket from './Ticket.js';
+import Button from '@material-ui/core/Button';
+import './Profile.scss';
+
+
+class Profile extends React.Component {
+  state = {
+    boughtSeats: true
+  }
+
+  changeSessions = () => {
+    this.setState({boughtSeats: !this.state.boughtSeats})
+  }
+
+  activeSessions = () => {
+    let bookSeats = [];
+    this.props.boughtSeatsByUser.map(bookSeat => { 
+      if(new Date().getTime() < bookSeat.session.date) {
+        bookSeats.push(bookSeat)
+      } 
+    })
+    return bookSeats;
+  }
+
+  completedSessions = () => {
+    let boughtSeats = [];
+    this.props.boughtSeatsByUser.map(bookSeat => { 
```

Comment:
you don't need map, because you don't edit the elements of the array 
it should be foreach